utilities/EluvioLiveCli.js

```diff
@@ -639,11 +639,13 @@ const CmdTenantWallets = async ({ argv }) => {
 
     if (argv.csv && argv.csv != "") {
       console.log(`CSV: ${argv.csv}`);
-      let out = "user_address,ident,created,extra_json\n";
+      let out = "user_address,ident,created,extras\n";
       for (let i = 0; i < res.contents.length; i++) {
         const ident = res.contents[i].ident ? res.contents[i].ident : "";
-        const json = res.contents[i].extra_json ? JSON.stringify(res.contents[i].extra_json) : "";
-        out = out + res.contents[i].addr + "," + ident + "," + res.contents[i].created + "," +  json + "\n";
+        let json = res.contents[i].extra_json ? JSON.stringify(res.contents[i].extra_json) : "";
+        json = json.replaceAll("\"", "'")
```

Comment:
technically in CSV a double quote can be escaped by doubling it,  not sure whether anyone is taking the csv and parsing it back as a json.

Reply:
https://www.rfc-editor.org/rfc/rfc4180.txt page 2 item 7 

7.  If double-quotes are used to enclose fields, then a double-quote
       appearing inside a field must be escaped by preceding it with
       another double quote.  For example:

       "aaa","b""bb","ccc"

Reply:
sure i can do that